modules/betweenBidAdapter.md

```diff
@@ -0,0 +1,29 @@
+# Overview
+
+Module Name: Example Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: prebid@example.com
+
+# Description
+
+Module that connects to Example's demand sources
+
+# Test Parameters
+```
+    var adUnits = [
+        {
+            code: 'test-div',
+            bids: [
+                {
+                    bidder: "between",
+                    params: {
+                        placementId: '12345',
```

Comment:
is 12345 an actual bid parameter?

Reply:
I updated pull req and i did pull req in prebid.io